hydra-node/src/Hydra/Network/Ouroboros.hs

```diff
@@ -187,7 +228,7 @@ withOuroborosNetwork localHost remoteHosts networkCallback between = do
    where
     responder =
       MuxPeer
-        showStdoutTracer
+        nullTracer
```

Comment:
Shouldn't we pass a proper tracer with `contramap Something` ?

Reply:
The `Mux` tracers are very noisy and very little useful. So I've completely turn them off (as we did in the past, but we only turned one side off).